antipatterns/mega-service.json

```diff
@@ -25,7 +25,12 @@
         "@inproceedings{palma2015study, title={A study on the taxonomy of service antipatterns}, author={Palma, Francis and Mohay, Naouel}, booktitle={Patterns Promotion and Anti-patterns Prevention (PPAP), 2015 IEEE 2nd Workshop on}, pages={5--8}, year={2015}, doi={10.1109/PPAP.2015.7076848}}"
       ],
       "tags": [
-        "architecture", "system", "service", "many_methods", "low_cohesion"
+        "microservices",
+        "soa",
+        "architecture",
+        "system",
+        "many_methods",
+        "cohesion"
```

Comment:
Maybe we should add the tag `abstraction` since it is mentioned in multiple antipatterns.
E.g. Client Completes Service, Chatty Service, Megaservice, Nano Service, Scattered Parasitic Functionality,..